docs/rst/user_manual/application_menu.rst

```diff
@@ -111,6 +111,13 @@ monitor, they will no longer be available in the whole application framework.
 The latter means that it will not be possible to plot charts with data relating to these entities.
 This button hides/reveals the currently inactive entities detected by the monitor.
 
+Hide/Show Metatraffic
+---------------------------
+Entities utilized for sharing metatraffic data are not shown by default. These include Fast-DDS Statistics module
```

Comment:
NIT: Use `.` to add line breaks.
Try to not break sentences. And when it is mandatory, try to break them in the "softest" way